integration-cli/docker_cli_logs_test.go

```diff
@@ -305,3 +305,14 @@ func (s *DockerSuite) TestLogsWithDetails(c *check.C) {
 	c.Assert(details[0], checker.Equals, "baz=qux")
 	c.Assert(details[1], checker.Equals, "foo=bar")
 }
+
+func (s *DockerSuite) TestLogsWithTag(c *check.C) {
```

Comment:
`integration-cli` is deprecated, we should not be adding any more tests to this suite (#34407 is pending to document this).

Unit tests should be sufficient for testing this change.

Reply:
I see it in the docs now. I'll have this backed out.